lib/apiservers/engine/backends/vicbackends.go

```diff
@@ -114,3 +124,40 @@ func ProductVersion() string {
 func VchConfig() *config.VirtualContainerHostConfigSpec {
 	return vchConfig
 }
+
+func createImageStore() error {
+
+	// TODO(jzt): we should move this to a utility package or something
+	host, err := sys.UUID()
+	if host == "" {
+		host, err = os.Hostname()
+	}
```

Comment:
This condition should never happen.  It's safe to remove 132:134 and just check for and return err if not nil.